apps/src/javalab/javalabRedux.js

```diff
@@ -100,6 +104,22 @@ export const setIsRunning = isRunning => ({
   isRunning
 });
 
+/**
+ * We should move isStartMode and levelName into a separate level redux file,
```

Comment:
It'd be great to create a JIRA ticket for this work if we want to do it at some point!

Reply:
Good call- documented here and will add to the pr description: https://codedotorg.atlassian.net/browse/CSA-578